annotation-server/src/guidelines/guidelines.controller.ts

```diff
@@ -26,4 +54,17 @@ export class GuidelinesController {
             dto.orderby ?? 'asc',
         );
     }
+
+    @ApiOperation({ summary: 'Fetch one guideline' })
+    @ApiParam({
+        name: 'id',
+        description: 'ID of the guideline to fetch information for',
+        example: '144',
+        type: 'integer',
+        required: true,
+    })
```

Comment:
I've addressed this in [86515a1](https://github.com/hpi-dhc/PharMe/pull/361/commits/86515a14278c15d7eda0f41c96419992b45943b3) in the context of #361 since this is where the issue first appeared and rebased this branch accordingly.